vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt

```diff
@@ -321,19 +321,15 @@ class CreateRoomViewModel @AssistedInject constructor(
 //                            preset = CreateRoomPreset.PRESET_PRIVATE_CHAT
 //                        }
 //                    }
-                    // Disabling federation
-                    disableFederation = state.disableFederation
-
-                    // Tchap: Use custom encrypt rules for Tchap, depends on which type of room is chosen
-                    // Encryption
-                    val shouldEncrypt = when (state.roomJoinRules) {
-                        // we ignore the isEncrypted for public room as the switch is hidden in this case
-                        RoomJoinRules.PUBLIC -> false
-                        else                 -> state.isEncrypted ?: state.defaultEncrypted[state.roomJoinRules].orFalse()
-                    }
-                    if (shouldEncrypt) {
-                        enableEncryption()
-                    }
+//                    // Encryption
+//                    val shouldEncrypt = when (state.roomJoinRules) {
+//                        // we ignore the isEncrypted for public room as the switch is hidden in this case
+//                        RoomJoinRules.PUBLIC -> false
+//                        else                 -> state.isEncrypted ?: state.defaultEncrypted[state.roomJoinRules].orFalse()
+//                    }
+//                    if (shouldEncrypt) {
+//                        enableEncryption()
+//                    }
 
                     if (state.roomType == TchapRoomType.EXTERNAL) {
```

Comment:
I think we can move the following lines (until the federation config) above the commented bloc to have a single bloc dedicated to tchap custom configuration. Eventually, we can move the tchap config to a dedicated method to facilitate the rebases